src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile

```diff
@@ -40,6 +40,10 @@ ENV DOTNET_DbgMiniDumpName="/dumps-share/coredump.%p"
 
 EXPOSE 5001
 
+# configure adress sanitizer
+ENV ASAN_OPTIONS='detect_leaks=0'
+ENV LD_PRELOAD=/usr/bin/../lib/gcc/x86_64-linux-gnu/12/libasan.so
```

Comment:
Why not just? 
```suggestion
ENV LD_PRELOAD=/usr/lib/gcc/x86_64-linux-gnu/12/libasan.so
```

Reply:
I copied it from a command output and did not bother simplifying the path.